frontend/packages/operator-lifecycle-manager/integration-tests-cypress/tests/deprecated-operator-warnings.cy.ts

```diff
@@ -0,0 +1,166 @@
+    cy.byTestID('source-community-operators-for-testing-deprecation').click();
+
+    cy.log('filter by the operator name');
+    cy.byTestID('search-operatorhub').type(testOperatorName);
+    cy.get('.co-catalog-tile').its('length').should('be.gt', 0);
+
+    cy.log('verify the Kiali Community Operator with a Deprecated badge');
+    cy.get('.co-catalog-tile').contains('.catalog-tile-pf-title', testOperatorName).should('exist');
+    cy.get('.co-catalog-tile')
+      .contains('.catalog-tile-pf-subtitle', deprecatedBadge)
+      .should('exist');
+    cy.get('.co-catalog-tile').click();
+
+    cy.log('verify the Deprecated badge on Install Operator modal');
+    cy.byTestID('deprecated-operator-warning-badge').contains(deprecatedBadge);
+
+    cy.log('verify the channel dropdown is displayed');
+    cy.get('.co-operator-channel__select').should('exist');
+
+    cy.log('click the channel dropdown to open and click the deprecated operator channel');
+    cy.get('.co-operator-channel__select').first().click({ force: true });
+    cy.get('.pf-v5-c-select__toggle').children().eq(1).click({ force: true });
+    cy.get('.pf-v5-c-select__menu-item').eq(1).click({ force: true });
+
+    cy.log('click the version dropdown to open and click the deprecated operator version');
+    cy.get('.co-operator-version__select').first().click({ force: true });
+    cy.get('.pf-v5-c-select__toggle').eq(1).click({ force: true });
+    cy.get('.pf-v5-c-select__menu-item').eq(15).click({ force: true });
+
+    cy.log(
+      'verify that the deprecated massages for package, channel and version are displayed on the install operator modal.',
+    );
+    cy.byTestID('deprecated-operator-warning-package').contains(deprecatedPackageMessage);
+    cy.byTestID('deprecated-operator-warning-channel').contains(deprecatedChannelMessage);
+    cy.byTestID('deprecated-operator-warning-version').contains(deprecatedVersionMessage);
+
+    cy.log('click the Install button on the Install Operator modal');
+    cy.byLegacyTestID('operator-install-btn').click({ force: true });
+
+    cy.log('verify the deprecated badge on Install Operator details page');
+    cy.byTestID('deprecated-operator-warning-badge').contains(deprecatedBadge);
+
+    cy.log('verify that the deprecated operator channel icon is displayed');
+    cy.byTestID('deprecated-operator-warning-channel-icon').should('be.visible');
+
+    cy.log('verify that the deprecated operator version icon is displayed');
+    cy.byTestID('deprecated-operator-warning-version-icon').should('be.visible');
+
+    cy.log(
+      'verify that the deprecated messages for package, channel, and version are displayed on the install operator modal.',
+    );
+    cy.byTestID('deprecated-operator-warning-package').contains(deprecatedPackageMessage);
+    cy.byTestID('deprecated-operator-warning-channel').contains(deprecatedChannelMessage);
+    cy.byTestID('deprecated-operator-warning-version').contains(deprecatedVersionMessage);
```

Comment:
Assert that these are visible
```suggestion
    cy.byTestID('deprecated-operator-warning-package').contains(deprecatedPackageMessage).should('be.visible');
    cy.byTestID('deprecated-operator-warning-channel').contains(deprecatedChannelMessage).should('be.visible');
    cy.byTestID('deprecated-operator-warning-version').contains(deprecatedVersionMessage).should('be.visible');
```